examples/message_demo/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "examples_rclrs_message_demo"
-version = "0.2.0"
+version = "0.3.0"
```

Comment:
Please revert the version changes, we'll make these changes when we prepare for the 0.3.0 release. Thanks.

Reply:
Reverted!